packages/generator-single-spa/src/react/templates/react.package.json

```diff
@@ -2,7 +2,9 @@
   "name": "<%= name %>",
   "scripts": {
     "start": "webpack serve",
+    "start:https": "webpack serve --env https",
```

Comment:
Part of what I'm hoping to accomplish is to document how it's possible to run both http and https. Many organizations end up using both because the root config runs over http but deployed environments are https, so they use http when working on root config and another service, but https otherwise.